pkg/capabilities/consensus/ocr3/requests/handler.go

```diff
@@ -152,12 +151,9 @@ func (h *Handler) expirePendingRequests(ctx context.Context) {
 
 	for _, req := range h.pendingRequests {
 		if now.After(req.ExpiresAt) {
```

Comment:
Can this also be simplified now? i.e. can we just have this logic inline in the Execute call since we're no longer yielding back to the caller?

Reply:
I'd prefer to do that simplification as a second PR as this one is quite beastly already

Reply:
raised a JIRA against me for this-> Can we simplify request handler in OCR now the call in execute call is sync?